pallets/substrate-app/src/lib.rs

```diff
@@ -358,6 +369,19 @@ pub mod pallet {
             )?;
             Ok(())
         }
+
+        /// Linits amount of tokens to transfer
+        /// WARNING - the limit should be without decimals, it should be in absolute value
+        #[pallet::call_index(5)]
+        #[pallet::weight(<T as Config>::WeightInfo::register_erc20_asset())]
```

Comment:
Is it correct? Weight as `register_erc20_asset`?

Reply:
There is a task to update weights.

Reply:
created task https://github.com/sora-xor/sora2-network/issues/473